ts/data/sample.ts

```diff
@@ -646,11 +654,57 @@ export function addMetricValueContents(
   }
 }
 
+function createCollapseButton(
+  contentWrapper: HTMLElement,
+  toggleText: string,
+  shouldCollapse: boolean
```

Comment:
This is only called when you want to collapse, and with an explicit value of `true`, so there's no need for the parameter